Xamarin.Forms.Controls/GalleryPages/OSThemesGalleryXaml.xaml

```diff
@@ -0,0 +1,55 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<ContentPage xmlns="http://xamarin.com/schemas/2014/forms" xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml" xmlns:gallerypages="clr-namespace:Xamarin.Forms.Controls.GalleryPages" x:Class="Xamarin.Forms.Controls.GalleryPages.OSThemesGalleryXaml">
+    <ContentPage.Resources>
+        <AppThemeColor x:Key="MyColor" Light="HotPink" Dark="Yellow" />
+        <Style x:Key="OSThemeStyle" TargetType="Label" >
+            <Setter Property="TextColor" Value="{OnAppTheme Black, Light=Green, Dark=Red}" />
+        </Style>
+        <StyleSheet>
+            <StyleSheetExtension.Style>
+                <OnAppTheme x:TypeArguments="x:String">
+                    <OnAppTheme.Light>
+                        <![CDATA[
+                            #cssStyledLabel {
+                                color: purple;
+                            }
+                        ]]>
+                    </OnAppTheme.Light>
+                    <OnAppTheme.Dark>
+                        <![CDATA[
+                            #cssStyledLabel {
+                                color: orange;
+                            }
+                        ]]>
+                    </OnAppTheme.Dark>
+                </OnAppTheme>
+            </StyleSheetExtension.Style>
+        </StyleSheet>
+    </ContentPage.Resources>
+    <ScrollView>
+        <StackLayout HorizontalOptions="Center" VerticalOptions="Center">
+            <Label TextColor="{OnAppTheme Light=Green, Dark=Red}">This text is green or red depending on Light (or default) or Dark</Label>
+            <Label>
+                <Label.TextColor>
+                    <OnAppTheme x:DataType="Color" Light="Green" Dark="Red" />
```

Comment:
Should the OnAppTheme extension detect theme changes update dynamically?

Reply:
Ideally yes, just not sure how. Could be something for a next iteration

Reply:
Implemented some code that should make the runtime updating experience better